fix(content): Apply styling to Signin TOTP

vpomerleau opened the pull request:
## Because

* Sign in with TOTP screen did not have styling applied to header and subheader

## This pull request
* Add styling to sign_in_totp_code mustache file

## Issue that this pull request solves

Closes: #FXA-8559

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Before:
![image](https://github.com/mozilla/fxa/assets/22231637/195a9542-9f15-4005-8d35-d330a1620436)

After:
![image](https://github.com/mozilla/fxa/assets/22231637/78e7228a-3ef9-4517-bf6a-f9e002bd92bd)

## Other information (Optional)

Any other information that is important to this pull request.

vbudhram reviewed (approved):
@vpomerleau In triage, we mentioned that another page had this issue too. r+ with that fix.

vpomerleau commented:
Thanks @vbudhram! I've updated the PR with the additional view that was mentioned in FXA-8565, and updated a few more that I found by searching for the "service" class.
